model/adjuster/clockskew.go

```diff
@@ -53,6 +51,10 @@ const (
 	warningFormatInvalidParentID = "invalid parent span IDs=%s; skipping clock skew adjustment"
 )
 
+var (
+	hostKeyTags = []string{"jaeger.hostname", "ip"}
```

Comment:
also should add `"hostname"`, it's used in [Go client](https://github.com/jaegertracing/jaeger-client-go/blob/master/constants.go#L36)